test/builder.test.ts

```diff
@@ -12,4 +16,11 @@ describe('builder', function() {
   it('appends middleware koa');
   it('prepends middleware koa');
   it('uses port');
+
+  it('should get app name from config', () => {
```

Comment:
🙃  can you add a test where options.appName is set?

Reply:
Done